### PR TITLE
fix pip install error

### PR DESCRIPTION
**Pull request by JaerongA**
- fixed an error in importing the module.
- updated Readme.
- added `.gitignore` to ignore .egg-info after installation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+# Cython debug symbols
+cython_debug/
+
+# PyCharm
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+#.idea/
```
Comment (kabilar) [inline .gitignore]: Thanks @JaerongA.  What repo is this `.gitignore` based off of?  Should we use the `.gitignore` that we have been using for the Elements?

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @JaerongA.  One question above.

**Comment by tdincer** [comment]
I don't think this PR is necessary. The file is not needed to be in a module.

**Comment by tdincer** [comment]
@kabilar Could you try to install the repo and import the load_file? I don't experience the bug.

**Comment by JaerongA** [comment]
> I don't think this PR is necessary. The file is not needed to be in a module.

hmm encountering this error after installation.

image

**Comment by tdincer** [comment]
> > I don't think this PR is necessary. The file is not needed to be in a module.
> 
> hmm encountering this error after installation.
> 
> image

Could you try loading the package in a different terminal? I'm just trying to understand the nature of the bug.

File: .gitignore (hunk 1 above)
Comment (JaerongA) [reply .gitignore]: @kabilar  I took it from here [https://github.com/github/gitignore/blob/main/Python.gitignore](url)  but yes, it makes sense to use one from element-repo

**Comment by tdincer** [comment]
@kabilar @JaerongA  I don't understand why this is not a bug on my machine. After you guys resolve the `.gitignore` conflict, let's merge this PR. Chris also couldn't import the function.

**Review by tdincer — APPROVED** [review]

**Pull request merged**